package-lock.json

```diff
@@ -0,0 +1,2249 @@
+{
```

Comment:
if possible, are you OK to use pnpm instead of npm, and remove this file.

since the pnpm-lock here is to keep the consitency of dependencies version. 

It is not good to have 2 different lock file in the repo.